pwiz_tools/Skyline/Model/DocSettings/Modification.cs

```diff
@@ -175,6 +176,8 @@ public StaticMod(string name, string aas, ModTerminus? term, LabelAtoms labelAto
         [Track]
         public bool Label13C { get { return (LabelAtoms & LabelAtoms.C13) != 0; } }
         [Track]
+        public bool Label14C { get { return (LabelAtoms & LabelAtoms.C14) != 0; } }

```

Comment:
I think, let's not have this shorthand, as we do not have a shorthand for labeling all O as 17O. We only have the shorthand for labeling all O as 18O.